build/utils/dotnet/solution/solution.go

```diff
@@ -140,8 +140,8 @@ func (solution *solution) GetProjects() []project.Project {
 	return solution.projects
 }
 
-func (solution *solution) DependenciesSourcesExist() bool {
-	return len(solution.dependenciesSources) > 0
+func (solution *solution) DependenciesSourcesAndProjectsPathExist() bool {
+	return len(solution.dependenciesSources) > 0 && len(solution.projects) > 0
```

Comment:
How about having two distinct functions? Each could verify the presence of different elements.

Reply:
I deleted the function and added a separate getter func GetDependenciesSources